src/screens/CreateTokenSymbol.js

```diff
@@ -38,28 +39,49 @@ class CreateTokenSymbol extends React.Component {
     this.props.navigation.navigate('CreateTokenAmount', { name, symbol: this.state.symbol });
   }
 
+  isButtonDisabled = () => {
+    if (!this.state.symbol) {
+      return true;
+    }
+
+    if (this.state.symbol.length < 2) {
+      return true;
+    }
```

Comment:
What about the max length of 5?

Reply:
We already forbid that in the input field, bu I'll change to also enforce here.

Reply:
Done